File: src/main/java/com/summer/lijiahao/utils/openapi/OpenApiToolWin.java
package com.summer.lijiahao.utils.openapi;

import com.intellij.openapi.project.Project;
import com.intellij.openapi.wm.ToolWindow;
import com.intellij.openapi.wm.ToolWindowFactory;
import com.intellij.ui.content.Content;
import com.intellij.ui.content.ContentFactory;
import org.jetbrains.annotations.NotNull;

public class OpenApiToolWin
        implements ToolWindowFactory {
    public void createToolWindowContent(@NotNull Project project, @NotNull ToolWindow toolWindow) {
        OpenApiView openApiView = new OpenApiView();
        ContentFactory contentFactory = ContentFactory.getInstance();
        Content content = contentFactory.createContent(openApiView, "", true);
        toolWindow.getContentManager().addContent(content);
    }
}

File: src/main/kotlin/com/summer/lijiahao/nccdevtool/opanapi/OpenApiToolWin.kt
package com.summer.lijiahao.nccdevtool.opanapi

import com.intellij.openapi.project.Project
import com.intellij.openapi.wm.ToolWindow
import com.intellij.openapi.wm.ToolWindowFactory
import com.intellij.ui.content.ContentFactory

class OpenApiToolWin : ToolWindowFactory {
    override fun createToolWindowContent(project: Project, toolWindow: ToolWindow) {
        val openApiView = OpenApiView()
        val contentFactory = ContentFactory.getInstance()
        val content = contentFactory.createContent(openApiView, "", true)
        toolWindow.contentManager.addContent(content)
    }
}